Contributors.md

```diff
@@ -1,11 +1,12 @@
 <!---
 This file is generated using the generate-contributors.py script. DO NOT MANUALLY EDIT!!!!
-Last Modified: 2016-11-23 08:32
+Last Modified: 2017-03-27 11:42
 --->
 
 The following people have contributed to the SCAP Security Guide project
 (listed in alphabetical order):
 
+* 0x66656c6978 <0x66656c6978@users.noreply.github.com>
```

Comment:
@0x66656c6978 we'd like to include you in the contributors list. Since we don't actually know your name, we entered it as 0x66656c6978 ;) Let us know if you'd like it to be changed to something else!